rio/src/org/usfirst/frc/team5806/robot/CameraClient.java

```diff
@@ -0,0 +1,78 @@
+import java.net.*;
+import java.io.*;
+
+public class CameraClient {
+	private String serverName;
+	private INetAddress staticIP;
+	private int serverPort;
+	private Socket client;
+	private BufferedWriter out;
+	private BufferedReader in;
+	
+	public CameraClient(String serverName, int serverPort) {
+		this.serverName = serverName;
+		this.serverPort = serverPort;
+	}
+	public CameraClient(String staticIP, int serverPort) {
```

Comment:
pretty sure you can't have two methods in java with the same method signature (that is the same name + the same types of params in the same order)